app/_layout.tsx

```diff
@@ -77,13 +81,20 @@ function RootLayoutNav() {
 
   return (
     <ThemeProvider value={colorScheme === 'dark' ? DarkTheme : DefaultTheme}>
-      <AuthProvider>
-        <Stack>
-          <Stack.Screen name="index" options={{headerShown: false}} />
-          <Stack.Screen name="(auth)" options={{headerShown: false}} />
-          <Stack.Screen name="(user)" options={{headerShown: false}} />
-        </Stack>
-      </AuthProvider>
+      <StatusBar
```

Comment:
Is there any reason we are changing the status bar within this pull request?

Reply:
Statusbar is used to show the icons on top of the phone (wifi, notification, battery life, etc.) when using the app. I just added it in for that. Is that bad?